uint32[] pose_index_list
---
ram_msgs/AdditiveManufacturingPose[] poses
